version.py

```diff
@@ -4,4 +4,4 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-__version__ = "2.2.0a1"
+__version__ = "3.1.0a1"
```

Comment:
This is necessary to avoid compatibility [check](https://github.com/googleapis/python-spanner-django/blob/196c44949370fb818e610b21c3b00344fdc3d03a/django_spanner/utils.py#L6) failure.